concepticondata/conceptlists/Morucci-2019-643.tsv

```diff
@@ -0,0 +1,644 @@
+Morucci-2019-643-389	389	ottagonale	octagonal	0.051	0	4.026	0	4.821	4.821	vista	8.897	0.721	0.542	4.824	4.944	4.722		
+Morucci-2019-643-390	390	ovale	oval	0.051	0.075	4.077	0.359	4.923	4.923	vista	9.485	0.894	0.514	5.882	4.167	5.722		
+Morucci-2019-643-391	391	pacifico	peaceful	0.154	0.237	0.333	3.205	3.564	3.564	vista	7.493	1.044	0.455	6.588	3.833	5.944		
+Morucci-2019-643-392	392	paffuto	chubby	0.05	0.103	4.175	0.525	4.625	4.625	vista	9.478	0.948	0.483	5.294	3.778	5.5		
+Morucci-2019-643-393	393	paonazzo	crazy	0	0	0.575	0.3	4.925	4.925	vista	5.8	0.521	0.849	4.353	5.059	4.944	1899	MAD
+Morucci-2019-643-394	394	parallelo	parallel	0.079	0.026	3.256	0.105	4.744	4.744	vista	8.211	0.803	0.575	5.316	4.588	5.737		
+Morucci-2019-643-395	395	pastoso	mellow	3.5	0.667	3.775	0.45	3.675	3.775	tatto	12.067	1.367	0.276	5	5.056	3.889		
+Morucci-2019-643-396	396	patinato	coated	0.179	0.128	3.385	0.513	4.282	4.282	vista	8.487	1.026	0.489	3.842	5.588	4.947		
+Morucci-2019-643-397	397	peloso	furry	0.615	0.385	4.795	0.692	4.744	4.795	tatto	11.231	1.174	0.393	6.111	3.35	6.368		
+Morucci-2019-643-398	398	pensato	thought	0.41	0.308	0.462	1.947	1.795	1.947	udito	4.922	1.337	0.333	6.421	3.765	4.389		
+Morucci-2019-643-399	399	pentagonale	pentagonal	0.026	0.026	4.231	0.103	4.974	4.974	vista	9.359	0.777	0.529	2.722	5.1	4.158		
+Morucci-2019-643-400	400	pepato	peppery	4.895	3.5	0.231	0.158	1.692	4.895	gusto	10.476	1.164	0.452	3.842	4.647	5.421		
+Morucci-2019-643-401	401	perimetrico	perimetric	0.051	0.103	2.897	0.333	4.436	4.436	vista	7.821	0.914	0.561	2.211	5.294	3.421		
+Morucci-2019-643-402	402	perpendicolare	perpendicular	0.051	0	3.436	0.077	4.947	4.947	vista	8.511	0.755	0.581	6.059	5	5.889		
+Morucci-2019-643-403	403	personale	personal	0.744	1.282	1.308	2.179	3.205	3.205	vista	8.718	1.491	0.282	6.941	3.667	5.5		
+Morucci-2019-643-404	404	pesante	heavy	0.697	0.706	4.441	2	3.765	4.441	tatto	11.609	1.375	0.323	6.947	2.941	6.579	1210	HEAVY
+Morucci-2019-643-405	405	pestifero	pestiferous	0.436	1.051	0.282	2.632	3.308	3.308	vista	7.709	1.285	0.393	3.789	3.235	4.842		
+Morucci-2019-643-406	406	pestilenziale	pestilential	1.579	3.842	0.342	0.324	2.684	3.842	olfatto	8.772	1.281	0.401	4.176	5.562	4.389		
+Morucci-2019-643-407	407	piagnucolante	whining	0	0.054	0.324	4.919	4.054	4.919	udito	9.351	0.847	0.526	4.529	3.667	5.5		
+Morucci-2019-643-408	408	piangente	crying	0.294	0.176	1.059	4.059	4.727	4.727	vista	10.316	1.129	0.441	4.647	3.833	5.722		
+Morucci-2019-643-409	409	piatto	dish	0.641	0.513	4.59	0.769	4.615	4.615	vista	11.128	1.221	0.369	6.882	3	6.611		
+Morucci-2019-643-410	410	piccante	spicy	5	2.528	0.444	0.083	1.944	5	gusto	10	1.191	0.492	6.526	3.588	6.526	544	SPICY
+Morucci-2019-643-411	411	piccino	small	0.553	0.316	4.103	1.154	4.923	4.923	vista	11.048	1.215	0.417	5.333	3.2	6.211	1246	SMALL
+Morucci-2019-643-412	412	piccolo	small	0.564	0.231	4.615	1.205	4.949	4.949	vista	11.564	1.191	0.408	7	2.235	6.579		
```

Comment:
If there must be a single link to SMALL, this is far more common than "piccino" (which is more likely used for "small kid")